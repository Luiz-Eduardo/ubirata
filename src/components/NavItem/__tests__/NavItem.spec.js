import '@testing-library/jest-dom';
import { render } from '@testing-library/vue';
import NavItem from '../NavItem.vue';

const renderComponent = (options) => render(NavItem, {...options });

const route = { fullPath: '/', path: '/' };

vi.mock('vue-router', () => ({
  useRoute: () => route,
  RouterLink: () => 'Home'
}));

describe('NavItem', () => {
  it('renders correctly', () => {
    const { container } = renderComponent({
      props: {
        to: '',
        iconSrc: '',
        iconAltText: '',
        title: ''
      }
    });

    expect(container).toMatchSnapshot();
  });
});
