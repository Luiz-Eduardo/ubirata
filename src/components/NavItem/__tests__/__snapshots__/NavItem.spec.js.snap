// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NavItem > renders correctly 1`] = `
<div>
  <li
    class="list-none"
  >
    Home
  </li>
</div>
`;
